spec/models/invitation_spec.rb

```diff
@@ -78,5 +78,17 @@
         end
       end
     end
+
+    context "Sending an invitation to a valid email" do
+      let(:invite) { build(:invitation, invitee_email: "foo@example.com") }
+
+      it "succeeds and 'sent_at' is set" do
+        expect(invite.save).to be_truthy
+        expect(invite.id).not_to be_nil
+
+        invite.reload
+        expect(invite.sent_at).not_to be_nil
```

Comment:
Note that `reload` returns the same record for chaining, so you can write this as `expect(invite.reload.sent_at).not_to be_nil`.

Reply:
I'm usually not a fan of being that terse (because it makes large tests really tough to read), but with a test this small I think it does make sense.